staging/src/k8s.io/apiextensions-apiserver/examples/client-go/hack/update-codegen.sh

```diff
@@ -25,7 +25,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-"${CODEGEN_PKG}/generate-groups.sh" all \
+bash "${CODEGEN_PKG}/generate-groups.sh" all \
```

Comment:
just curious - is bash mandatory here?

Reply:
this is coming from this community commit: https://github.com/kubernetes/kubernetes/pull/79081/commits/391da79abd7a7af1cfab04de35daba7cd58c4c80